### PR TITLE
Feauture  adding mental illness variables in fact art

### PR DESCRIPTION
**Pull request by Marymary-dev**

### CODE REVIEW

File: Scripts/NDWH/C&T FACT TABLES/load_FactART.sql [hunk 1]
```diff
@@ -1,4 +1,28 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
```
Comment (cbrianbet) [inline Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:4]: Kinldy remove the merge errors

Comment (Marymary-dev) [reply Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:4]: @cbrianbet ,done

Comment (nobert-mumo) [reply Scripts/NDWH/C&T FACT TABLES/load_FactART.sql:4]: not sure what happened, but I thought we already had this @Marymary-dev  ?

**Comment by Marymary-dev** [comment]
yeah, am not sure @chestor brought this to my attention

**Comment by Marymary-dev** [comment]
Already exists

**Pull request closed without merging**